pkgs/development/python-modules/mat2/default.nix

```diff
@@ -47,6 +47,11 @@ buildPythonPackage rec {
     ./executable-name.patch
     # hardcode path to mat2 executable
     ./tests.patch
+    # fix gmenu path
```

Comment:
Comments should be updated to something “Fix gobject-introspection typelib path for Nautilus extension”